src/Thumbnail.php

```diff
@@ -140,10 +140,13 @@ public function adminPostThumbnailHtml(string $content, int $postId, $attachment
             $removeImageLabel = $postTypeObject->labels->remove_featured_image;
         }
 
-        return $this->replaceRemovePostThumbnailMarkup(
+        $content = $this->replaceRemovePostThumbnailMarkup(
             $removeImageLabel,
             $content
         );
+
+        return $this->site->replaceGlobalUrlPath($content);
```

Comment:
I understanding this is the main fix for the PR but I haven't enough informations from your PR comment, could you help in reproduce the problem? May be by adding the steps to reproduce it?